app/src/main/java/org/sopt/daangnmarket_android/ui/view/main/HomeFragment.kt

```diff
@@ -0,0 +1,62 @@
+package org.sopt.daangnmarket_android.ui.view.main
+
+import android.content.Intent
+import android.os.Bundle
+import androidx.fragment.app.Fragment
+import android.view.LayoutInflater
+import android.view.View
+import android.view.ViewGroup
+import org.sopt.daangnmarket_android.databinding.FragmentHomeBinding
+import org.sopt.daangnmarket_android.ui.view.data.ItemData
+import org.sopt.daangnmarket_android.ui.view.main.adapter.ItemAdapter
+import org.sopt.daangnmarket_android.ui.view.write.WriteActivity
+
+class HomeFragment : Fragment() {
+    private lateinit var itemAdapter: ItemAdapter
+    private var _binding : FragmentHomeBinding? = null
+    private val binding get() = _binding!!
```

Comment:
```suggestion
    private val binding get() = _binding ?: error("binding not initialized")
```

!!을 쓰면 _binding이 null일 경우 NullPointerException이 발생할 수 있습니다. 이 경우 예상치 못한 앱의 비정상 종료가 일어나기 때문에 문제가 될 수 있습니다. 그래서 `?:` 를 사용해 null일 경우에 대한 예외처리를 하는게 좋을 것 같습니다.